spec/requests/likes/create_spec.rb

```diff
@@ -29,6 +30,28 @@
     end
   end
 
+  context 'Success' do
```

Comment:
Please write a better context name for this spec such as `Comment` or `Liking a Comment`. It would be more descriptive that way.